docs/executor/index.rst

```diff
@@ -30,6 +30,10 @@ full path to the module e.g. ``my_acme_company.executors.MyCustomExecutor``.
 
 .. toctree::
     :maxdepth: 1
-    :glob:
 
-    *
+    sequential
```

Comment:
Why did you change it? Are there any problems with glob?

Reply:
I am actually on the fence here. With `glob`, both Sequential and Local executors will sort to the bottom. But for documentation, I think it will be better to sort the executors based on the complexity and powerfulness of the executors. However, it adds a bit of overhead.  What do you think?

Reply:
Fine. It makes sense.